example-project/apigw-app/Main.hs

```diff
@@ -4,7 +4,6 @@ module Main where
 import           AWSLambda.Events.APIGateway
 import           Control.Lens
 import qualified Data.HashMap.Strict         as HashMap
-import           Data.Semigroup
```

Comment:
This will probably break on older GHC versions - Not sure if adding a CPP #if is the correct way to deal with that or not, but I'd how I've done it in the past.